6-async/01-callbacks/article.md

```diff
@@ -171,19 +170,20 @@ loadScript('/my/script.js', function(error, script) {
 });
 ```
 
-Once again, the recipe that we used for `loadScript` is actually quite common. It's called the "error-first callback" style.
+Una vez más, la receta que usamos para `loadScript` es bastante usada. Es llamado estilo "error-first callback".
 
-The convention is:
-1. The first argument of the `callback` is reserved for an error if it occurs. Then `callback(err)` is called.
-2. The second argument (and the next ones if needed) are for the successful result. Then `callback(null, result1, result2…)` is called.
+La conveción es:
 
-So the single `callback` function is used both for reporting errors and passing back results.
+1. El primer argumento del callback `callback` es reservado para un error en caso que ocurra. Entonces se llama`callback(err)`.
+2. El segundo argumento (y los siguientes si se necesita) son para el resultado exitoso. Entonces se llama `callback(null, result1, result2…)`.
 
-## Pyramid of Doom
+De esta manera esta única función `callback` es usada para ambas cosas, reportar errores y pasar de vuelta los resultados.
 
-From the first look, it's a viable way of asynchronous coding. And indeed it is. For one or maybe two nested calls it looks fine.
+## Pirámide de la catástrofe
```

Comment:
creo que debería de agregarse entre paréntesis el termino en ingles ya que es muy usado por la comunidad y serviría de referencia en ingles. 

(Pyramid of Doom)